### PR TITLE
probe: return dirty and shutdown status despite metadata mismatch

### PR DESCRIPTION
**Pull request by arutk**

### CODE REVIEW

**Review by Ostrokrzew — DISMISSED** [review]

**Review by imjfckm — APPROVED** [review]

File: src/mngt/ocf_mngt_cache.c [hunk 1]
```diff
@@ -2050,7 +2064,7 @@ static void _ocf_mngt_cache_unplug(ocf_cache_t cache, bool stop,
 	__deinit_cleaning_policy(cache);
 	__deinit_promotion_policy(cache);
 
-	if (ocf_mngt_cache_is_dirty(cache)) {
+	if (dirty) {
```
Comment (robertbaldyga) [inline src/mngt/ocf_mngt_cache.c:2067]: Why can't we just do this earlier (before removing cores)?

Comment (arutk) [reply src/mngt/ocf_mngt_cache.c:2067]: we can, but it seems to belong here logically - don't you think?

Comment (robertbaldyga) [reply src/mngt/ocf_mngt_cache.c:2067]: Imo we could even do it as a first thing during cache stop. We don't expect any flushing at that point anyway.

**Review by robertbaldyga — APPROVED** [review]

**Review by jfckm — APPROVED** [review]

**Comment by arutk** [comment]
retest this please

**Pull request merged**